lib/http/cookie_jar/abstract_store.rb

```diff
@@ -122,3 +111,17 @@ def cleanup(session = false)
     # self
   end
 end
+
+require 'http/cookie_jar/hash_store'
+
+begin
+  require 'http/cookie_jar/mozilla_store'
+rescue LoadError
```

Comment:
could we instead avoid `LoadError` if we know *mozilla_store.rb* is not compatible w JRuby?

Reply:
Yes, I can rework this.

Reply:
addressed in 5a71a8bf0021a6a54b0f31fcfc7a5369a168217e.